app/Views/index.twig

```diff
@@ -47,19 +47,10 @@
 		  	<div class="sidebar content-box" style="display: block;">
                 <ul class="nav">
                     <!-- Main menu -->
+                    {% if (data.role != 'Employee' and data.role != 'Manager') %}
```

Comment:
what if we will have 10 roles in the future? please spend a bit more time to think on how to improve your current approach